frontend/src/hooks/query/use-active-port.ts

```diff
@@ -0,0 +1,46 @@
+import { useQueries, useQuery } from "@tanstack/react-query";
+import axios from "axios";
+import React from "react";
+import { openHands } from "#/api/open-hands-axios";
+import {
+  useWsClient,
+  WsClientProviderStatus,
+} from "#/context/ws-client-provider";
+
+export const useActivePort = () => {
+  const { status } = useWsClient();
+  const [activePort, setActivePort] = React.useState<string | null>(null);
+
+  const { data } = useQuery({
+    queryKey: ["ports"],
```

Comment:
I think we need to return `hosts`, not `ports`, if this is going to work with remote runtime

Reply:
It does indeed return the host, I just forgot to rename it 😅